src/van.cc

```diff
@@ -253,6 +253,16 @@ void Van::Start(int customer_id) {
     // get my node info
     if (is_scheduler_) {
       my_node_ = scheduler_;
+      std::string ip;
+      const char *itf = Environment::Get()->find("DMLC_INTERFACE");
+      std::string interface;
+      if (itf) interface = std::string(itf);
+      if (interface.size()) {
+          GetIP(interface, &ip);
+      } else {
+          GetAvailableInterfaceAndIP(&interface, &ip);
+      }
+      my_node_.hostname = ip;
```

Comment:
By this way, we alwasy use the ip instead of the hostname. Will it affect other use cases?

Reply:
sorry, I only check local cases, I will try [https://github.com/dmlc/ps-lite/pull/129](https://github.com/dmlc/ps-lite/pull/129)